src/citrine/informatics/modules.py

```diff
@@ -30,6 +30,4 @@ class Module(PolymorphicSerializable['Module'], AsynchronousObject):
     def get_type(cls, data) -> Type['Module']:
         """Return the subtype."""
         from citrine.informatics.design_spaces import DesignSpace
-        return {
```

Comment:
Bleh, this must have felt nice to update.

Reply:
I'm so excited for 3.0, when we can rip it all out completely! =D